.github/workflows/foundation-ang-trigger.yml

```diff
@@ -0,0 +1,47 @@
+# Controls when the action will run. The workflow runs when manually triggered using the UI or API
+
+name: Foundation_Ang_On_Demand
+
+on:
+  workflow_dispatch:
```

Comment:
Can we have one single file for on dispatch and on push ?

Reply:
I added the push event to check the workflow, it will be removed after testing the manual foundation and trigger on the master.

Reply:
Why don't add another trigger instead of adding another workflow?

Reply:
Done